pkg/featureSet/osdctl/osdctl.go

```diff
@@ -14,11 +15,18 @@ func New(home string) (*Config, error) {
 
 	config := &Config{}
 
-	config.Mounts = append(config.Mounts, engine.VolumeMount{
-		Source:       home + "/" + osdctlConfigDir,
-		Destination:  "/root/" + osdctlConfigDir,
-		MountOptions: "ro",
-	})
-
+	config.Mounts = append(
+		config.Mounts,
+		engine.VolumeMount{
+			Source:       home + "/" + osdctlConfigDir,
+			Destination:  "/root/" + osdctlConfigDir,
+			MountOptions: "ro",
+		},
+		engine.VolumeMount{
```

Comment:
This will need to be hidden behind a conditional in case the file does not exist on the host system.

I suggest something like the following pseudocode:

```
if os.Stat(home + "/" + vaultTokenFile) {
  config.Mounds = append(
    config.Mounts,
    engine.VolumeMount{
      Source:       home + "/" + vaultTokenFile,
	  Destination:  "/root/" + vaultTokenFile,
	  MountOptions: "rw",
    })
}
```

Reply:
fixed up in [34ed777](https://github.com/openshift/ocm-container/commit/34ed777d25db8328c405eaf39cf031897581a61c)